rts/lra/lra-test/src/test/java/io/narayana/lra/participant/SpecIT.java

```diff
@@ -168,14 +178,14 @@ public void finishTest() throws Exception {
 
     @Test
     public void startLRA() throws WebApplicationException {
-        URL lra = lraClient.startLRA("SpecTest#startLRA", LRA_TIMEOUT_MILLIS);
+        URL lra = lraClient.startLRA(null, "SpecTest#startLRA", LRA_TIMEOUT_MILLIS, TimeUnit.SECONDS);
```

Comment:
I can't help but feel there could be a startLRA operation which doesn't require you to pass null as the first param for the parentLRA